docs/content/users/configuration/hooks.md

```diff
@@ -143,24 +151,22 @@ hooks:
     - exec: "drush cc all"
 ```
 
-## Drupal 8 Example
+## Drupal 10 Example
```

Comment:
I'm not sure, but isn't "Drupal 8 Example" still relevant for Drupal 8, and "Drupal 10 Example" might have its own section?

Reply:
There was command differences between Drupal 7 and Drupal 8 (eg: drush cc all to drush cr) but D8 onwards the commands are the same.

Drupal.org doesn't list Drupal 8 or 9 as current versions (D8 was end of life in 2021 and D9 was November 2023)
so it was just to keep with consistency with the Project.

Reply:
Okay, that sounds good.